dev-libs/libtypec/libtypec-0.5.1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
```

Comment:
I see that there's also a `meson.build` upstream. I would suggest using that instead of cmake as it's a much lighter dependency and will almost certainly already be installed since meson is used to build portage itself.

At the same time, I see that the version in the project's `meson.build` is wrong:


```
conf_data = configuration_data()
conf_data.set('libtypec_VERSION_MAJOR', '0')
conf_data.set('libtypec_VERSION_MINOR', '5')
conf_data.set('libtypec_VERSION_PATCH', '0')
```

Reply:
Well, looks like the meson build system is also broken, so there's that.

Also, it builds `libtypec.so.1` whereas cmake builds `libtypec.0.5.1`